src/print.jl

```diff
@@ -370,6 +354,38 @@ end
 ## Constraints
 #------------------------------------------------------------------------
 
+function show_constraints_summary(io::IO, model::Model)
+    for (F, S) in MOI.get(model, MOI.ListOfConstraints())
+        num_constraints = MOI.get(model, MOI.NumberOfConstraints{F, S}())
+        println(io, "`$F`-in-`$S`: $num_constraints constraint",
+                plural(num_constraints))
+    end
+    if !iszero(num_nl_constraints(model))
+        println(io, "Nonlinear: ", num_nl_constraints(model), " constraint",
+                plural(num_nl_constraints(model)))
+    end
+end
+
+function constraints_string(print_mode, model::Model, sep, eol)
+    str = ""
+    for (F, S) in MOI.get(model, MOI.ListOfConstraints())
+        for idx in MOI.get(model, MOI.ListOfConstraintIndices{F, S}())
+            # FIXME the shape may be incorrect here
```

Comment:
Does an issue need to be opened for this?

Reply:
Done: https://github.com/JuliaOpt/JuMP.jl/issues/1723